components/infobox/wikis/fighters/infobox_league_custom.lua

```diff
@@ -198,7 +193,115 @@ function CustomLeague.getIconFromSeries(page)
 	if not series then
 		return
 	end
+
 	return series.icon, series.icondark
 end
 
+function CustomLeague:_createPrizepool()
+	if String.isEmpty(_args.prizepool) and
+		String.isEmpty(_args.prizepoolusd) then
+		return nil
+	end
+
+	local localCurrency = _args.localcurrency
+	local prizePoolUSD = _args.prizepoolusd
+	local prizePool = _args.prizepool
+
+	if localCurrency == 'text' then
```

Comment:
needed for fighters?
this is only on sc2 because there are pages having plain text in the ppt display

if you do not need checking for text you can drop this if branch and also clean up the other one

Reply:
Removed text handling and hasText related code